slyd/slyd/jsonapi/registry.py

```diff
@@ -0,0 +1,16 @@
+from slyd.orm.exceptions import ImproperlyConfigured
+
+
+__all__ = [
+    'schema',
+]
+
+schemas = {}
+
+
+def get_schema(schema_type):
```

Comment:
Change this to `get_serializer` or `registry_serializer` for better indicating we are caching serializers in the registry.